src/seedu/addressbook/AddressBook.java

```diff
@@ -354,7 +342,8 @@ public static String executeCommand(String userInputString) {
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
         default:
-            return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
```

Comment:
The rest are not quite, though. You are merely adding a new line ;)

Remember that it's a logical sense of abstraction.